<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>线程启动UNO，9600，端口内置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>主程序启动　mega2560，9600，端口内置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>读取端口</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_main">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_th">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lint_main_send">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_main_send">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lint_th_send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_th_send">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
